content/outcomes/secure-software-supply-chain/container-registry.md

```diff
@@ -0,0 +1,33 @@
+---
+title: How a Container Registry Can Help
+weight: 6
+layout: single
+---
+
+A container registry provides a single place to store and retrieve your container images. At a minimum, a container registry will support the [Docker Registry API](https://docs.docker.com/registry/spec/api/), the API that both Docker and Kubernetes expect to be supported to pull container images. They may either be a publicly hosted such as [Docker Hub](https://hub.docker.com/) or hosted privately such as [VMware Harbor](https://goharbor.io/).
```

Comment:
Should this refer to the OCI distribution spec more generally? (than specifically the Docker registry?) - https://github.com/opencontainers/distribution-spec

Reply:
Also maybe GCR is a better example than DockerHub? (given the ratelimiting in effect and seemingly huge migration away from DHub?)